common/pom.xml

```diff
@@ -2,65 +2,74 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>io.brooklyn.clocker</groupId>
-    <artifactId>clocker-parent</artifactId>
-    <version>2.1.0-SNAPSHOT</version> <!-- CLOCKER_VERSION -->
-  </parent>
+    <parent>
+        <groupId>io.brooklyn.clocker</groupId>
+        <artifactId>clocker-parent</artifactId>
+        <version>2.1.0-SNAPSHOT</version> <!-- CLOCKER_VERSION -->
+    </parent>
 
-  <name>Clocker :: Common</name>
-  <artifactId>common</artifactId>
-  <packaging>jar</packaging>
+    <name>Clocker :: Common</name>
+    <artifactId>clocker-common</artifactId>
+    <packaging>jar</packaging>
 
-  <build>
-    <resources>
-      <resource>
-        <directory>src/main/resources</directory>
-        <filtering>false</filtering>
-      </resource>
-      <resource>
-        <directory>src/test/resources</directory>
-        <filtering>false</filtering>
-        <targetPath>tests</targetPath>
-      </resource>
-    </resources>
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/examples</directory>
```

Comment:
What purpose does moving the examples folder up a level serve? 
Will it not end up in the same place after build?